feats/selector.py

```diff
@@ -88,20 +97,22 @@ def _hex_hash(self, key: str) -> str:
         ).hexdigest()
 
     def select(self, value: object) -> str:
-        key = self.segment(value)
+        key = self.segment.segment(value)
```

Comment:
So just to clarify, the first `segment` in `self.segment.segment(value)` is the capital "S" `Segment` object (noun), and the second `segment` is the verb segment (i.e. we're segmenting based on `value`), right?

Not trying to harp too hard on this - I know you've already mentioned possibly renaming segments anyways.

Reply:
Yepppp 😬